test/helpers/application_helper_test.rb

```diff
@@ -0,0 +1,8 @@
+require 'test_helper'
+
+class ApplicationTestHelperTest < ActionView::TestCase
+    test 'full title helper' do
+        assert_equal full_title, 'Ruby on Rails Tutorial Sample App'
+        assert_equal full_title("Help"), 'Help | Ruby on Rails Tutorial Sample App'
```

Comment:
If you are feeling lucky perhaps look up the Faker gem to generate text for tests?